src/medCore/medGlobal.cpp

```diff
@@ -0,0 +1,8 @@
+#include <medGlobal.h>
+#include <QCryptographicHash>
+
+QString anonymise(const QString name) {
+    QByteArray r = QCryptographicHash::hash(name.toUtf8(),QCryptographicHash::Md4).toHex().toUpper();
```

Comment:
Technically this doesn't generate a unique id per patient name if I'm not mistaken. I know the probability of two names generating the same hash is very low but just to make sure, what will happen if two patients are anonymised to the same name ?